### PR TITLE
Adjust Whitelist template

### PR DESCRIPTION
**Pull request by kisak-valve**
- Drop a couple pointless line wraps
- Request distro version instead of kernel version
- Adjust the no workarounds used line

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/whitelist-request.md [hunk 1]
```diff
@@ -11,21 +11,19 @@ about: Games tested and found to have no issues.
 ## System Information
 - GPU: <!-- e.g. RX 580 or GTX 970 -->
 - Driver/LLVM version: <!-- e.g. Mesa 18.2/7.0.0 or nvidia 396.54 -->
-- Kernel version: <!-- e.g. 4.17 -->
+- Distro version: <!-- e.g. 4.17 -->
```
Comment (dreamer) [inline .github/ISSUE_TEMPLATE/whitelist-request.md]: Comment with example should be probably changed, maybe to "Ubuntu 18.10".

Comment (kisak-valve) [reply .github/ISSUE_TEMPLATE/whitelist-request.md]: Thanks!

**Comment by aeikum** [comment]
Thanks, will merge.

**Comment by kisak-valve** [comment]
Thanks for landing in 67307a91df3458dc60b2c7d476cfd5e78ace79f5.

**Pull request closed without merging**